src/main/java/org/generationcp/breeding/manager/listmanager/util/GermplasmListExporter.java

```diff
@@ -346,34 +346,17 @@ protected GermplasmList getGermplasmListAndListData(final Integer listId) throws
 
 		final Map<Integer, Variable> variableMap = new HashMap<>();
 		this.addVariableToMap(variableMap, TermId.SEED_AMOUNT_G.getId());
-		final Variable stockId = createMockStockId();
-		variableMap.put(stockId.getId(), stockId);
-		return variableMap;
-	}
 
-	//	TODO: This method was created to continue export the stockId after removing him FROM cvterm.
-	// 	IBP-4058 Remove StockID variable from the ontology
-	private Variable createMockStockId() {
+		//	TODO: This variable was created to continue exporting the stockId after removing him FROM cvterm.
+		// 	IBP-4058 Remove StockID variable from the ontology
 		final Variable variable = new Variable();
 		variable.setName(ColumnLabels.STOCKID.getName());
-		variable.setDefinition("ID of an inventory deposit");
-		final Property property = new Property();
-		property.setName("GERMPLASM STOCK ID");
-		variable.setProperty(property);
-		final Scale scale = new Scale();
-		scale.setName("Germplasm id");
-		scale.setDefinition("Germplasm scale to hold Germplasm List data type");
-		scale.setDataType(DataType.GERMPLASM_LIST);
-		variable.setScale(scale);
-		final Method method = new Method();
-		method.setName("Assigned");
-		method.setDefinition("Term, name or id assigned");
-		variable.setMethod(method);
-		variable.addVariableType(VariableType.GERMPLASM_DESCRIPTOR);
-		variable.setId(8269);
-		return variable;
+		variable.setId(-8269);
+		variableMap.put(-8269, variable);
```

Comment:
Can we at least use the enum with negative value?

Reply:
I think this fix is gonna be for a short period and it's used just only in this place. what do you think @clarysabel ?

Reply:
I agree with @nahuel-soldevilla 
It is not only used here but in Commons, I do not want to spend much time fixing this because we need to reimplement the export but at least if the number is in only one place willbe easier to identify it later when doing the new implementation